### PR TITLE
[terra-framework-docs] Fix slide panel changelog

### PR DESCRIPTION
**Pull request by trandrew1023**
### Summary

**What was changed:**
Latest merge to slide panel duplicated the changelog, updated so the latest is in unreleased

**Why it was changed:**
Fix for changelog and release documentation

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-framework-docs/CHANGELOG.md [hunk 1]
```diff
@@ -16,8 +13,8 @@
 ## 1.26.0 - (July 20, 2023)
 
 * Added
+  * Added examples for `terra-worklist-data-grid` when using pinned columns.
```
Comment (sdadn) [inline packages/terra-framework-docs/CHANGELOG.md:16]: This should be under unreleased.

Comment (sdadn) [reply packages/terra-framework-docs/CHANGELOG.md:16]: Actually, this should be fixed by #1720 , so let's wait for that to get merged and see if there's any remaining changes to be made.

Comment (artpark) [reply packages/terra-framework-docs/CHANGELOG.md:16]: @sdadn @trandrew1023 Merged

Comment (trandrew1023) [reply packages/terra-framework-docs/CHANGELOG.md:24]: Checked out the latest and it looks good to me, closing this PR. Thanks!

**Pull request closed without merging**